include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -130,18 +126,18 @@ struct SPhysicalDeviceLimits
     uint64_t nonCoherentAtomSize = 0ull;
 
     /* VkPhysicalDeviceSparseProperties */ 
-    //VkBool32    residencyStandard2DBlockShape;
-    //VkBool32    residencyStandard2DMultisampleBlockShape;
-    //VkBool32    residencyStandard3DBlockShape;
-    //VkBool32    residencyAlignedMipSize;
-    //VkBool32    residencyNonResidentStrict;
+    //bool    residencyStandard2DBlockShape;
+    //bool    residencyStandard2DMultisampleBlockShape;
+    //bool    residencyStandard3DBlockShape;
+    //bool    residencyAlignedMipSize;
+    //bool    residencyNonResidentStrict;
 
 
 
 
     /* Vulkan 1.1 Core  */
-    uint32_t subgroupSize;
-    core::bitflag<asset::IShader::E_SHADER_STAGE> subgroupOpsShaderStages;
+    uint32_t subgroupSize = 0u;
+    core::bitflag<asset::IShader::E_SHADER_STAGE> subgroupOpsShaderStages = asset::IShader::E_SHADER_STAGE(0u);
```

Comment:
Prefer `ESS_UNKNOWN` rather than setting enum to 0u.
this is only in the cases where we don't have 0 value in the enums. we usually have something with the name UNKNOWN, DEAULT, etc for the enum 0 value